### ISSUE TITLE
Default FPP config will not allow eth0 and wlan0 connections at the same time and inhibits FPP hotspot upon initial boot

### ISSUE THREAD

**Issue by reelpilot** [FalconChristmas/fpp#1646]
FPP 7.x running on Pi 3B+ and Pi 4's

**Describe the bug and Steps to reproduce**: 
I discovered this issue when reimaging my pi's for the upcoming season. Upon boot up after reimaging AND having the pi connected via ethernet to a controller or colorlight card, the pi sees that connection and will not respond with the wifi hotspot to allow for initial configuration via wifi. If the ethernet cable is unplugged and the pi rebooted, then the wifi hotspot will come up, however all wifi connectivity terminates when the cable is plugged back into eth0. This can be fixed by going into the advanced settings and changing the etho priority from 0 to 1. 

**Expected behavior**: 
This behavior differs from previous versions of FPP and the steps to correct the issue and get everything working may not be intuitive for newer users. Anyone who is running their FPP instance wirelessly may encounter this problem when first booting FPP 7 and has it plugged into their controller. Allowing the wifi hotspot to come up on initial boot when FPP is plugged into a controller may help newer users as well as additional language to make adjusting the priority setting clearer for newer users.

**Comment by bloojoop** [comment]
the tethering mode has always been "if no connection", this is not new.. the behavior I have always see (using fpp since 2.0) has been if the ethernet is connected, the tethering interface will not start up.  Also if using both interfaces, you should not have IP address in the same subnet if if ethernet connects to colorlight card should not have an IP at all, if connecting to a 'separate show network' then it should NOT have a gateway defined.

**Comment by reelpilot** [comment]
Every controller I have is paired with its own pi and is run as a remote. The same is true for each colorlight. I have always been able to take a controller or colorlight away from my show environment, like at our monthly meetings, power it on and it will broadcast the FPP hotspot. It will not do that anymore because it's connected to the controller. This is new behavior and not how previous versions operated. I can verify this at home by simply disabling my show network and booting up a pi on v7 and I get no FPP hotspot. However if I do the exact same thing with a pi running v5.x that I have yet to update, it will broadcast the FPP hotspot. Both eth0 and wlan0 are set to completely different subnets on the controller installations and only wlan0 is configured on the colorlights.

**Comment by bloojoop** [comment]
I certainly will defer to the actual devs for comment/issue.  

I have deployed several hundred fpp instances now.  The default behavior has always been 'if no connection' which means tethering will not start if a eth link is detected.  This can be changed to always be available so tethering is always on (I did two of those this week for standalone installs) but then you cannot configure it as a client to join a wifi AP.  when you say disabled the show network, not sure the method used but if that 'disconnects' the eth, like a port disable on the switch, then I would expect the tethering to start up upon reboot.    ( I just tested this on one of my testpi installs)  .
all that said, if the eth0 side is configured correctly, then plugging it in should NOT change the wifi communication if connected to an AP.
